python/startup/bootstrap.py

```diff
@@ -212,8 +211,9 @@ def _ensure_extension_up_to_date():
         installed_version = installed_version_file.read().strip()
 
     if installed_version is None:
-        raise Exception(
-            "Could not determine version for the installed extension.")
+        logger.debug("Could not determine version for the installed extension. Reinstalling")
```

Comment:
minor: suggest consistency with previous debug message with regard to `, reinstalling` vs. `. Reinstalling`. I like this form better so suggestion changing the one above.